libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -196,6 +196,21 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("POS2", 17, AP_GPS, _antenna_offset[1], 0.0f),
 
+    // @Param: DELAY_MS
+    // @DisplayName: GPS delay in milliseconds
+    // @Description: Controls the amount of GPS  measurement delay that the autopilot compensates for. Set to zero to use the default delay for the detected GPS type.
+    // @Units: msec
```

Comment:
I did a search through the code found the following usages in different places

milli-seconds
milliseconds
Milliseconds
msec (EKF libraries)

I will add patches to change them all to milliseconds